src/components/pages/inventory/setup/item/CreateItem.tsx

```diff
@@ -1,81 +1,250 @@
-import { useCreateItemMutation } from '../../../../../components/service/itemsApi';
-import { Button, Form, Input, Modal, Select } from 'antd';
+import {
+  Button,
+  Col,
+  DatePicker,
+  Form,
+  Input,
+  Modal,
+  Row,
+  Select,
+  Upload,
+  message,
+} from 'antd';
+import { UploadOutlined } from '@ant-design/icons';
 import React from 'react';
+import { useCreateItemMutation } from '../../../../../service/itemsApi';
 const { Option } = Select;
 
 interface CreateModalProps {
   title: string;
   visible: boolean;
   onCancel: () => void;
-  onSave: (newData: any) => void;
 }
 
 const CreateItem: React.FC<CreateModalProps> = ({
   title,
   visible,
   onCancel,
-  onSave,
 }) => {
   const [form] = Form.useForm();
   const [createItem] = useCreateItemMutation();
 
-  const handleAddItem = async (values: any) => {
+  const onFinish = async (values: any) => {
```

Comment:
Please remove `any` from here and add type

Reply:
You can follow this one https://github.com/bablukpik/inventory-hasura-graphql/blob/main/src/components/Products/AddProduct.tsx#L15C42-L15C42